pyartcd/pyartcd/pipelines/build_sync.py

```diff
@@ -392,10 +397,16 @@ async def build_sync(runtime: Runtime, version: str, assembly: str, publish: boo
         exclude_arches=exclude_arches,
         skip_multiarch_payload=skip_multiarch_payload
     )
-
+    span = trace.get_current_span()
+    span.set_attributes({
+        "pyartcd.param.dry_run": runtime.dry_run,
+        "pyartcd.param.version": version,
+        "pyartcd.param.assembly": assembly,
+    })
     try:
         await pipeline.run()
         await pipeline.handle_success()
+        span.set_status(trace.StatusCode.OK)
 
     except (RuntimeError, ChildProcessError):
```

Comment:
Wouldn't it be worthy to `span.set_status(trace.StatusCode.ERROR)` here?

Reply:
This function is wrapped by a decorator, which will automatically set the result to trace.StatusCode.ERROR on exception.